js/lost_without_map.test.js

```diff
@@ -0,0 +1,14 @@
+const maps = require('./lost_without_map');
+
+
+describe('lost without map tests', () => {
+
+  test('Test-1 for simple multiplying 3 elements array', () => {
```

Comment:
The full test name does not read like a sentence

> lost without map tests Test-1 for simple multiplying 3 elements array

Please improve it :smile: You can use my other suggestions for test names